tests/tests.rs

```diff
@@ -1,18 +0,0 @@
-extern crate attestation;
```

Comment:
Keep this file, but rename it to mandos_go_test.rs
Also would be cool to copy the github actions from elrond-wasm.